### DISCUSSION TITLE
Is there any alternative to AnyBurn open-source?

### DISCUSSION THREAD

**Discussion by skhaz** [memstechtips/UnattendedWinstall#47] [General]
My main machine is a macOS. I can use Wine to run AnyBurn as I have done in the past, but I am wondering if there is a multiplatform alternative.

**Comment by memstechtips** [comment]
I'm not sure about an alternative to be honest. 
I've seen people saying that this works, maybe you can test it out? https://www.uubyte.com/iso-editor.html
